charts/tx-data-provider/Chart.yaml

```diff
@@ -23,12 +23,12 @@ name: tx-data-provider
 description: A Helm chart for Kubernetes
 
 type: application
-version: 0.0.5
+version: 0.0.6
 appVersion: 0.0.1
 
 dependencies:
   - name: digital-twin-registry
-    version: 0.4.5
+    version: 0.4.11
```

Comment:
@tunacicek I noticed that the [install of the tx-data-provider chart fails](https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/runs/9564181744/job/26364874640?pr=98#step:15:3324), maybe some configuration needs to be updated... is the edc version 0.5.3 even still compatible?

Reply:
I think in yesterday's sync we discussed that @ds-jhartmann would upgrade the dtr together with upload of test data script, right? So should this change be reverted?

Reply:
acutally, the reason why the tx-data-provider install fails is because of values-test-iam-init-container.yaml
The initContainer was still pointing to old paths so the centralidp never started which is required for the EDC.
I created the PR to fix this here: #103
With this fix, the DTR can be updated in this PR, as long as it is backwards compatible with 24.03

Reply:
thanks @ds-jhartmann https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/103 is approved

Reply:
I will move the changes for the DTR to the upgrade_2024_05 branchh -> because it is not backwards compatible